examples/data_augmentation/reinforcement/README.md

```diff
@@ -0,0 +1,82 @@
+## Reinforcement Learning based Data Augmentation for Text Classification
+
+This model builds upon the connection of supervised learning and reinforcement learning (RL), and adapts reward learning algorithm from RL for joint data augmentation learning and model training.  
+For details, please refer to the [Learning Data Manipulation for Augmentation and Weighting](https://arxiv.org/pdf/1910.12795.pdf) and the original [github](https://github.com/tanyuqian/learning-data-manipulation/).
+
+This example demonstrates the usage of `forte/models/da_rl/MetaAugmentationWrapper`, that wraps a BERT Masked Language Model data augmentation model to perform this RL adaptive learning with a BERT-based text classifier downstream model.
+
+
+### Examplge Usage
+
+See `RLAugmentClassifierTrainer.train_epoch()` in `main.py` for details.
+
```

Comment:
Can you briefly explain what happens here? Like, why we are wrapping, why we compute validation loss and real loss.